common/djangoapps/enrollment/tests/test_views.py

```diff
@@ -537,13 +547,13 @@ def test_deactivate_enrollment(self):
             )
 
         # Create a 'verified' enrollment
```

Comment:
Minor: This comment could be updated to read something like "Create an enrollment with the selected mode".